mudlet-mapper.xml

```diff
@@ -222,40 +222,38 @@ end</script>
             <Trigger isActive="yes" isFolder="no" isTempTrigger="no" isMultiline="no" isPerlSlashGOption="no" isColorizerTrigger="no" isFilterTrigger="no" isSoundTrigger="no" isColorTrigger="no" isColorTriggerFg="no" isColorTriggerBg="no">
                 <name>Alertness (Achaea+Imperian)</name>
                 <script>mmp.alertness = mmp.alertness or {}
-
+cecho(&quot;&lt;red&gt;\n&quot;..matches[4])
 local alertness = mmp.alertness
-
+ 
 local roomexists, getRoomName, anytolong, matches, deleteLine = mmp.roomexists, getRoomName, mmp.anytolong, matches, deleteLine
-
+ 
 if mmp.debug then
-  mmp.alertnessTime = mmp.alertnessTime or createStopWatch()
-  startStopWatch(mmp.alertnessTime)
+ mmp.alertnessTime = mmp.alertnessTime or createStopWatch()
+ startStopWatch(mmp.alertnessTime)
 end
-    
-
-for k, v in pairs (gmcp.Room.Info.exits) do
-  local longexitname = mmp.anytolong(k)
-  if roomexists(v) and matches[3]:lower() == getRoomName(v):lower() and longexitname then
-    alertness[longexitname] = alertness[longexitname] or {}
-    alertness[longexitname][#alertness[longexitname]+1] = matches[2]
-    deleteLine()
-
-   end
-end
-
+ 
+if matches[4] ~= &quot;location&quot; then
+ local longexitname = matches[4]
+ 
+ alertness[longexitname] = alertness[longexitname] or {}
+ alertness[longexitname][#alertness[longexitname]+1] = matches[2]
+ deleteLine()
+elseif matches[4] == &quot;location&quot; then
 local currentroomname = mmp.cleanroomname(mmp.currentroomname)
-
-if matches[3] == currentroomname then
-  alertness.here = alertness.here or {}
-  alertness.here[#alertness.here+1] = matches[2]
-  deleteLine()
+ 
+ if matches[3] == currentroomname then
+ alertness.here = alertness.here or {}
+ alertness.here[#alertness.here+1] = matches[2]
+ deleteLine()
```

Comment:
Considering @Ahmet already has it set up for the pull I sent the revised script to him to adjust. Easier that way considering I'm still trying to figure out how to work github stuff